packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,122 @@
+| `selected-dates`           | string[]                                                | -                 | selected dates                                                                                                                    |
+
+### Outputs
+
+- [selectedDates: string] - the selected dates from the calendar
+
+### Events
+
+- `change`: Emits when the selected date(s) is changed
+- `date-select`: Emits when user selects a date(s) when no date(s) is currently selected
+- `view-change`: Emits when view changes (e.g. from date picker to month picker) via user interaction
+
+### Slots
+
+- `start` - content before the calendar
+- `end` - content after the calendar
+- `link` - link at the bottom right of the base of the calendar (e.g. "Go to today")
+- `navicon-left` - left icon on header to navigate backwards for months, years, or range of years
+- `navicon-right` - right icon on header to navigate forwards for months, years, or range of years
+
+### CSS Variables
+
+None
+
+## Accessibility
+
+- [ ] Find the matching component through [WCAG's patterns](https://www.w3.org/WAI/ARIA/apg/patterns/)
+  - [Date Picker Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/)
+- [ ] Are there any accessibility elements unique to this component?
+- [ ] List ARIA attributes: `role, aria-labelledby, aria-label, aria-pressed, aria-selected, aria-live`
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [ ] Arrows - Moves focus according to the grid behavior
+  - [ ] Enter - Select the date
+  - [ ] Home - Moves focus to the first day (e.g. Sunday) of the current week
+  - [ ] End - Moves focus to the last day (e.g. Satuday) of the current week
+  - [ ] Page Up - Changes the grid of dates to the previous month
+  - [ ] Shift + Page Up - Changes the grid of dates to the previous year
+  - [ ] Page Down - Changes the grid of dates to the next month
+  - [ ] Shift + Page Down - Changes the grid of dates to the next year
+
+## Preparation
+
+- [x] This component will inherit from and document the [FAST Calendar Component](https://github.com/microsoft/fast/tree/master/packages/web-components/fast-foundation/src/calendar)
+
+- [x] [Check the Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/tree/master/specs) for differences and document:
+
+  - [Fluent UI React v8 Calendar Component Spec](https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar)
+  - [Fluent UI React v9 Date Picker Component Spec](https://github.com/microsoft/fluentui/blob/master/specs/Datepicker.md)
+
+  Differences
+
+  - The Calendar component in Fluent React v8 handles picking a range of dates differently from the v9 Date Picker component. The v9 Calendar should handle date ranges in the same way as the v9 Date Picker.
+  - Fluent UI has a Close button for the calendar, but the v9 Date Picker design spec does not
```

Comment:
Should the Calendar hide on ESC or mouse click outside of it?

Reply:
If we're using Combobox to present, it will be the Listbox that responds to the mouse and keyboard events for close.